contrib/python/src/python/pants/contrib/python/checks/tasks/python_eval.py

```diff
@@ -104,6 +104,9 @@ def _compile_target(self, target):
       if isinstance(target, PythonBinary):
         source = 'entry_point {}'.format(target.entry_point)
         components = target.entry_point.rsplit(':', 1)
+        if not all([x.strip() for x in components]):
+          raise TaskError('Invalid entry point {} for target {}'.format(
```

Comment:
It's clearly not a common or even likely error, or we'd have noticed it by now, so this is just a precaution, and the error message is probably fine. It would be laborious to explain in a comment, and I think whoever sees the actual broken entry_point string will immediately see why.